portfolio/src/main/webapp/script.js

```diff
@@ -28,24 +28,29 @@ function addRandomGreeting() {
   
 }
 
-function getdataUsingArrowFunctions() {
-  fetch('/data').then(response => response.json()).then((phrase) => {
+function ListComments() {
+  fetch('/formhandler').then(response => response.json()).then((CommentInfo) => {
 
     const commentCont = document.getElementById('commentCont');
     commentCont.innerText = '';
 
-    let increment = 1;
-    phrase.forEach((element) => {
+    CommentInfo.forEach((element) => {
+        let commentdiv = document.createElement('div');
 
-        let li = document.createElement('li');
-        li.innerHTML = ("Comment " + increment + ": " + element ); 
-        commentCont.appendChild(li);                 
-        console.log(element + "here");
-        increment++;
-    });
-    // let li = document.createElement('li');
-    // li.innerText = phrase;
-    // commentCont.appendChild(li);
+        let timestampS = document.createElement('p');
```

Comment:
It's Ok to construct elements from javascript for simple examples, but once you need to create complex structure this approach becomes unmaintainable. HTML's template and slot can help in this case: https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots

For this particular example using template would be an overkill.